Content/posts/internship-benny.md

```diff
@@ -0,0 +1,100 @@
+
+Another big factor was that the internship was entirely in Swift. Having just a semester of experience with Swift and having enjoyed using it, I wanted to get more experience in it.
+
+## The internship
+
+### Some statistics
+
+During my internship I've done:  
+- 30+ GitHub Issues  
+- 160+ Pull Requests  
+- 500+ Commits
+
+### Projects
+
+#### Blog website  
+  
+One of my first projects was working on the Blog site, the site you are on right now. This included some small things like adding a date and an author to the blog posts. But I've also added a 404 error page, automatic deployment to AWS by using GitHub Actions, adding CSP to the site and I've written a few blog posts.
+
+#### Documentation website  
+  
+On the documentation website, I've added a 404 page and deployed the website to AWS. Next to that I have also added the i18n plugin that makes translating for the documentation a smoother experience as we don't need to create multiple repositories anymore per language.
+
+#### Project boards  
+  
+One of the bigger project I've undertaken is the project boards. 
+The assignment was to create a GitHub Action workflow that, on creation of an issue with the label 'good first issue' or the label 'help wanted', would create a project card on the corresponding project board.
+
+To start this project off, I first went through the documentation of GitHub Actions to see which information we were getting on the creation of an issue. Following this it was a case of finding out how to create the project cards from this information. For this there was a marketplace action available made by [Alex Page](https://github.com/alex-page) that could automatically do this for us. Once we got this figured out, it was time to make the workflow reusable. This was needed as otherwise, we'd need to update every workflow in every repo that contains it if there's a mistake or change. For this, I've made a workflow that passes the labels on an issue to the reusable workflow. And behold, the project boards have come alive.
+
+Since this workflow has been in place, the 'good first issue' issues project board has been halfway completed.
+
+#### Template generation  
+  
+Another project entirely made in GitHub Actions that I've taken upon me was the automatic template creation. This was a suggestion made by Tim so people wouldn't have to install the toolbox to get a Vapor project created anymore. Overall this workflow was straightforward since the templates could be replaced easily. 
```

Comment:
```suggestion
Another project entirely made in GitHub Actions that I worked on was the automatic template creation. This was a suggestion made by Tim so people wouldn't have to install the toolbox to get a Vapor project created anymore. Overall this workflow was straightforward since the templates could be replaced easily. 
```